packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -71,6 +71,11 @@ function webpackConfigSingleSpa(opts) {
             },
           ],
         },
+        {
+          test: /\.html$/i,
```

Comment:
I think we can merge this in without it being a breaking change. It's possible that people have configured `.html` files to be loaded via something other than raw loader and that this causes problems for them when they upgrade. However, I don't think we can account for every possible user-specific change to this shared webpack config when evaluating whether something is breaking.